pkg/git/git_test.go

```diff
@@ -80,6 +81,24 @@ func TestPlainClone(t *testing.T) {
 	})
 }
 
+func TestPlainOpenMock(t *testing.T) {
+	t.Parallel()
+	t.Run("successful clone", func(t *testing.T) {
+		t.Parallel()
```

Comment:
What is the reason for the parallel statement here? There are no nested tests on that level. I _think_ that is not needed.

Reply:
Most/All tests are done like this:  TestPlainClone, TestChangeBranch,... Seemed to me like a pattern I should follow

Reply:
Ok, maybe we should investigate if that has any effect when there are no nested tests. But in case this is the same elsewhere that should be changed with another PR for all suspected occurances ...

Reply:
... and a valid point for having the Parallel() statements would be: now we don't have more tests on the same nesting level, but maybe in the future. Then we can introduce it as some kind of anticipating obedience. I personally would introduce it later when needed, but maybe it is also reasonable to do it like this ....